support/rewriteInheritance.js

```diff
@@ -0,0 +1,29 @@
+var through = require('through2');
+
+/**
+ * Replace all instances of `__proto__` in a string with a cross-browser
+ * compatible alternative.
+ *
+ * @param {string} js
+ * @return {string}
+ */
+function parseInheritance(js) {
+  return js.replace(/^ *(\w+)\.prototype\.__proto__ * = *(\w+)\.prototype *;?/gm, function(_, child, parent){
```

Comment:
Haha, no real effort involved, I just ported it from here:

https://github.com/mochajs/mocha/blob/master/support/compile.js#L59

But yeah, we should definitely fix it throughout. I just figured we could handle that as a separate issue and keep this PR's scope strictly to switching to Browserify, since it already runs the risk of breaking so many things